client/me/help/help-contact-closed/index.jsx

```diff
@@ -174,7 +174,14 @@ const HelpContactClosed = ( { translate, sitePlanSlug } ) => {
 
 	return (
 		<div className="help-contact-closed">
-			<FormSectionHeading>{ translate( 'Limited Support This Week' ) }</FormSectionHeading>
+			<FormSectionHeading>
+				{ translate( 'Limited Support %(closed_start_date)s - %(closed_end_date)s', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( '%(closed_start_date)s to %(closed_end_date)s' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).